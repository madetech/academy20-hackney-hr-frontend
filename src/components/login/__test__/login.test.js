import React from 'react';
import renderer from 'react-test-renderer';
import Login from '../login';

test('Login form is rendered', () => {
  const component = renderer.create(
    <Login />,
  );
  let test = component.toJSON();
  expect(test).toMatchSnapshot();
  
});