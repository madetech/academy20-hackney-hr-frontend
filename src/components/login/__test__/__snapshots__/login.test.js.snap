// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login form is rendered 1`] = `
<section>
  <span
    className="back"
  >
    <a
      href="/"
    >
       
      &lt;
       Back
    </a>
  </span>
  <div
    className="welcome-text"
  >
    <h1>
      Employee Login
    </h1>
    <p>
      Please enter your credentials.
    </p>
  </div>
  <form
    onSubmit={[Function]}
  >
    <label
      htmlFor="email"
    >
      Email:
    </label>
    <br />
    <input
      id="email"
      name="email"
      onChange={[Function]}
      type="text"
      value=""
    />
    <br />
    <label
      htmlFor="password"
    >
      Password:
    </label>
    <br />
    <input
      id="password"
      name="password"
      onChange={[Function]}
      type="password"
      value=""
    />
    <br />
    <input
      className="button-submit"
      type="submit"
      value="Submit"
    />
  </form>
</section>
`;
